SimTracker/SiPhase2Digitizer/plugins/PSPDigitizerAlgorithm.cc

```diff
@@ -68,3 +67,21 @@ void PSPDigitizerAlgorithm::accumulateSimHits(std::vector<PSimHit>::const_iterat
     ++simHitGlobalIndex;
   }
 }
+//
+// -- Select the Hit for Digitization
+//
+bool PSPDigitizerAlgorithm::select_hit(const PSimHit& hit, double tCorr, double& sigScale) {
+  bool result = false;
+  sigScale = 1.0;
```

Comment:
what's the purpose of this, since the variable is already set to 1 above (and then seemingly never used anywhere)?